modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/RoleResourceTest.java

```diff
@@ -225,16 +294,53 @@ public void testPostSiteRoleUserAccountAssociation() throws Exception {
 			assertHttpResponseStatusCode(
 				500,
 				roleResource.postSiteRoleUserAccountAssociationHttpResponse(
-					_getRoleId(_addRole(RoleConstants.TYPE_REGULAR)),
+					_getRoleId(_addRole(true, RoleConstants.TYPE_REGULAR)),
 					_user.getUserId(), testGroup.getGroupId()));
 			assertHttpResponseStatusCode(
 				500,
 				roleResource.postSiteRoleUserAccountAssociationHttpResponse(
-					_getRoleId(_addRole(RoleConstants.TYPE_ORGANIZATION)),
+					_getRoleId(_addRole(true, RoleConstants.TYPE_ORGANIZATION)),
 					_user.getUserId(), testGroup.getGroupId()));
 		}
 	}
 
+	@Override
+	@Test
+	public void testPutRoleByExternalReferenceCode() throws Exception {
```

Comment:
Why are we overriding this? It looks like as the super one

Reply:
there is small difference in the first line of the method:
Role postRole = testPutRoleByExternalReferenceCode_createRole();
vs
Role postRole = testPutRoleByExternalReferenceCode_addRole();